projects/drivers/util/util.c

```diff
@@ -144,3 +144,17 @@ uint32_t hweight8(uint32_t word)
 
 	return count;
 }
+
+/**
+ * Calculate the quotient and the remainder of an integer division.
+ */
+uint64_t do_div(uint64_t* n,
```

Comment:
should `do_div` be called `do_div64` ?
the idea is to make it clearer that it's u64, in case there would be a u32 `do_div` equivalent

Reply:
we can rename it to "do_div64" when a  "do_div32" is implemented?

Reply:
fine from my side